src/db/dbMigrator/dbMigrator.ts

```diff
@@ -0,0 +1,75 @@
+import * as path from 'path'
+// @ts-ignore
```

Comment:
is this line still needed?

Reply:
Yes , you added that actually... not sure why it doesn’t work